covert/__main__.py

```diff
@@ -70,6 +73,16 @@
 {shortdechelp}\
 """
 
+idhelp = """\
```

Comment:
Much more explanation is needed here to make "id" command understandable.

Reply:
Much more explanation added.